src/components/core/Main/Main.js

```diff
@@ -52,12 +55,20 @@ class Main extends Component {
     });
   };
 
+  onTextareaChange = ({target}) => {
+    this.setState({
+      TextareaValue: target.value,
```

Comment:
???

Reply:
I deleted it then and then added it again. Maybe trying to fix something. I am sorry